core.js

```diff
@@ -318,6 +318,13 @@ async function _fromTokenizer(tokenizer) {
 					};
 				}
 
+				if (zipHeader.filename.startsWith('3D/') && zipHeader.filename.endsWith('.model')) {
```

Comment:
```suggestion
				if (zipHeader.filename.startsWith('/3D/') && zipHeader.filename.endsWith('.model')) {
```

Reply:
There's no leading slash in the filenames of any of the zip-based formats. The above will fail.
xlsx has the same implementation: https://github.com/sindresorhus/file-type/blob/master/core.js#L314-L319